docs/firefox-accounts.rst

```diff
@@ -37,6 +37,18 @@ predictability in retroactive analysis.
     <https://mozilla.github.io/application-services/docs/accounts/metrics.html#descriptions-of-metrics-related-query-parameters>`_
     rules applied by the Firefox Accounts server.
 
+UITour Flow
+-----------
+
+Since Firefox 79 the FxA links and email forms use the UITour to show the Firefox Accounts page and log the browser
```

Comment:
@rfk anything else to mention here?

Reply:
nope, nothing in particular to add, thanks!